sklearn/cross_validation.py

```diff
@@ -973,9 +973,11 @@ def cross_val_predict(estimator, X, y=None, cv=None, n_jobs=1,
 
     cv : cross-validation generator or int, optional, default: None
         A cross-validation generator to use. If int, determines
-        the number of folds in StratifiedKFold if y is binary
-        or multiclass and estimator is a classifier, or the number
+        the number of folds in StratifiedKFold if y, or the number
```

Comment:
it should be "if y is a classifier", then you can remove the sentence about "StratifiedKFold" below.